doc/telemetry-dev-env/Readme.md

```diff
@@ -0,0 +1,46 @@
+Build the image:
+```
+docker build -t telemetry-dev .
+```
+
+Remove container if it exist already:
+```
+docker rm telemetry-dev-env
+```
+Create personal fork repo from sonic-telemetry from git
+
+Creatd directory 'src' and Clone git repos in it:
```

Comment:
`~/src` would be clearer. Because later (for docker run) on it is saying  `~/src`